perceval/backends/_mps.py

```diff
@@ -41,26 +40,36 @@
 
 
 class MPSBackend(AProbAmpliBackend):
-    """Step-by-step circuit propagation algorithm, works on a circuit.
-    Approximate the probability amplitudes with a cutoff.
+    """
+    The state of the system is written in form of an MPS and
+    updated step-by-step by a circuit propagation algorithm.
+
+    Approximate the probability amplitudes with a cutoff -> bond Dimension in an MPS.
     - For now only supports Phase shifters and Beam Splitters
```

Comment:
Shouldn't it be reworded to "supports components up to two modes", after what we've seen in the code?